app/assets/stylesheets/config/_fonts.scss

```diff
@@ -1,10 +1,28 @@
 // Import Google fonts
 @import url('https://fonts.googleapis.com/css?family=Nunito:400,700|Work+Sans:400,700&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;700&display=swap');
 
 // Define fonts for body and headers
-$body-font: "Roboto","Work Sans", "Helvetica", "sans-serif";
-$headers-font: "Roboto", "Nunito", "Helvetica", "sans-serif";
+$body-font: "Montserrat";
+$headers-font: "Montserrat";
```

Comment:
With the font change, I suggest leaving the backup font in after "Montserrat" like before